h/authService.h

```diff
@@ -28,6 +28,10 @@
 
 int installAuthCheckService(HttpServer *server);
 void installZosPasswordService(HttpServer *server);
+const char* getProfileNameFromRequest(char *profileName, char *url, char *method, int instanceID);
+int serveAuthCheck(HttpService *service, HttpResponse *res);
```

Comment:
TODO: serveAuthCheck doesn't need to be made public, but is here now (I was testing it) due to Leonty's request to see code earlier